pkg/sql/crdb_internal.go

```diff
@@ -1218,8 +1218,11 @@ j.error_msg,
 j.claim_instance_id
 FROM system.public.jobs AS j
 LEFT OUTER JOIN system.public.job_progress AS p ON j.id = p.job_id
-LEFT OUTER JOIN system.public.job_status AS s ON j.id = s.job_id  
-	` + whereClause
+LEFT OUTER JOIN system.public.job_status AS s ON j.id = s.job_id
+	` + whereClause + `UNION
+	(SELECT job_id, job_type, description, user_name, 'pending',
+					NULL, now(), NULL, now(), NULL, NULL, NULL, NULL
+	FROM crdb_internal.session_pending_jobs())`
```

Comment:
Should we be pushing the where clause into this sub-select as well?  I think that for a virtual table this works as-is because the virtual table will recheck each row against the predicate even if it is using an "index" generator.  But I think this would be wrong if this were a view?

Reply:
Right, this works as-is because there is a higher-level check in the virtual table machinery that verifies all returned rows match the predicate so it's fine to leave.

I could see fixing it now just because it is nice to be able to copy/pasta these queries directly into the terminal when debugging things. But whatever you prefer.